src/main/java/org/opentripplanner/middleware/utils/HttpUtils.java

```diff
@@ -24,6 +26,8 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(HttpUtils.class);
```

Comment:
Can be removed.

Reply:
Addressed in https://github.com/ibi-group/otp-middleware/pull/55/commits/6f19f22cccf1abe83b5d151ef4aa7888c711f726